Source/gg2/Objects/Alternative Objects/2014 Haxxy and Raybanns Rewards/Tombstone.events/Begin Step.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<event category="STEP" id="1">
+  <actions>
+    <action id="603" library="1">
+      <!--action name: Code-->
+      <kind>CODE</kind>
+      <allowRelative>false</allowRelative>
+      <question>false</question>
+      <canApplyTo>true</canApplyTo>
+      <actionType>CODE</actionType>
+      <functionName/>
+      <relative>false</relative>
+      <not>false</not>
+      <appliesTo>.self</appliesTo>
+      <arguments>
+        <argument kind="STRING">var mapHeight;
+mapHeight = map_height();
+
+if (floor(image_index) == (image_number - 1))
+    image_speed = 0;
+
+
+// Move down to meet the ground
```

Comment:
Oops, yes, I think in my excitement over the working code, I forgot to remove the dead stuff.